project_planning/README.rst

```diff
@@ -0,0 +1,46 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+================
+Project Planning
+================
+
+* This module extends the functionality of Project to support and allow you to reset the planning tasks week 1, 2, 3, and 4 to 0!
```

Comment:
This module extends the functionality of Project to support planning information:
* It creates the fields to display the work of period 1 to 4
* It allows you to reset the planning tasks week 1, 2, 3, and 4